app/models/rating.js

```diff
@@ -0,0 +1,39 @@
+// import dependencies
+const e = require('express')
```

Comment:
Isn't "e" short hand for "event"? Maybe it should be called something different here? What do you think?

Reply:
yep took it out it was automatically added